scripts/core/EXP-COMMAND-BUFFER.rst

```diff
@@ -132,6 +132,15 @@ were obtained from.
                                            pLocalWorkSize, 1, &syncPoint, 
                                            nullptr);
 
+    // Get SyncPoint profiling information
+    ${x}_event_handle_t event;
```

Comment:
think this example would be clearer if we set `event` to something valid, rather than left it uninitialized.